tests/integration/tests/test_sanity_checks.py

```diff
@@ -239,14 +244,18 @@ def test_cspm_azure_findings(cspm_client, match_type):
     Raises:
         AssertionError: If the resource type is missing.
     """
-    query_list = build_query_list(benchmark_id="cis_azure", version=AGENT_VERSION)
-    query, sort = cspm_client.build_es_must_match_query(
-        must_query_list=query_list,
-        time_range="now-24h",
-    )
+    azure_agents = wait_components_list(agents_actual_components, agents_expected_components, CIS_AZURE_COMPONENT)
+    for agent in azure_agents:
+        query_list = build_query_list(
+            benchmark_id="cis_azure",
+            version=AGENT_VERSION,
+            match_type=match_type,
+            agent=agent,
+        )
+        query, sort = cspm_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
```

Comment:
@amirbenun shouldn't we only check the agentless agent?

Reply:
The former revision of this function asserts for any findings of this benchmark, after installing CSPM Azure on the agentless agent, there are 2 agents running this benchmark. To verify they both produce findings, I have added the for loop to assert separately.